index.html

```diff
@@ -4,21 +4,46 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <div class="Container" id="mainContainer">
       <h1>Recipe App</h1>
-      <!-- car for display recipe-->
-      <div id="recipeCard">
-        <title>Name</title>
-        <div><img src="" alt="" /></div>
-        <div>
-          <h1>ingridiants</h1>
-          <div>
-            <table></table>
-          </div>
-        </div>
+      <!-- start card for display recipe-->
+      <div id="recipeCard"></div>
+      <!--End of card-->
+
+      <!-- start Form for recipe-->
+
+      <div id="recipeForm">
+        <h1>-- Your recipe--</h1>
+
+        <form action="">
+          <label for="recipeTitle">Recipe Name</label>
+          <input type="text" id="recipeTitle" name="title" required />
+          <label for="imageuplode">Picture</label>
+          <input type="file" id="imageuplode" name="uplode" />
+          <br /><br />
+          <fieldset id="fieldset">
+            <h1>Ingredients</h1>
+            <button type="button" id="bTnIngredients" required>
```

Comment:
Please name your IDs in pascalCase or kebab-case and be consistent with it

Reply:
Code has been refactored for good code practice.